iogt/templates/ga_tag_manager.html

```diff
@@ -0,0 +1,21 @@
+{% if settings.home.SiteSettings.local_ga_tag_manager %}
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ settings.home.SiteSettings.local_ga_tag_manager }}"
```

Comment:
This would trigger a request to an external domain. Hopefully, 'gtm.js' can be self-hosted, but what can we do about this iframe?

Reply:
could you check these articles
- https://developers.google.com/tag-platform/tag-manager/server-side/custom-domain
- https://stape.io/blog/how-to-add-a-custom-domain-to-the-google-tag-manager-server-container

these explain how to configure custom domain for google tag manager

Reply:
I'm finding it difficult to see how those two links address my question. My understanding is that we cannot have references to, for example, 'www.googletagmanager.com', so surely more work needs to be done here to make those references configurable.

Reply:
my understanding is that if we can configure custom domains (IoGT domains) as mentioned in the above articles then `gtm.js` and other assets could be served from those domains,
and making those base URLs configurable wouldn't take long, we can put those in site settings too